### DISCUSSION TITLE
API calls to OMCompiler

### DISCUSSION THREAD

**Discussion by albert-mathews** [OpenModelica/OpenModelica#10185] [Q&A] [answered]
With reference to the end of the conversation here:
https://github.com/OpenModelica/OpenModelica/issues/10151#issuecomment-1424449440

What API calls does OMEdit make? I would like my application to make API calls like OMEdit does, is this possible? will I find signatures for these in OMPython? if so, can you point me where to look for them? Does OMEdit makes calls to a library version of `omc`?

**Comment by adeas31** [comment, marked as answer]
You can find the API documentation here https://build.openmodelica.org/Documentation/OpenModelica.Scripting.html. Most of the API calls are documented but not all.

OMEdit does a lot of API calls and you can see them in the log file `%TEMP%/OpenModelica/OMEdit/omeditcommunication.log`. The log file contains the API calls with result. If you are only interested in seeing the list of API calls done by OMEdit then check `%TEMP%/OpenModelica/OMEdit/omeditcommands.mos`.
